Sources/SwiftyRequest/RestRequest.swift

```diff
@@ -625,6 +628,28 @@ public class RestRequest {
 
         return nil
     }
+
+    /// Method to identify the charset encoding defined by the Content-Type header
+    /// - Defaults set to .utf8
+    /// - Parameter contentType: The content-type header string
+    /// - Returns: returns the defined or default String.Encoding.Type
+    private func getCharacterEncoding(from contentType: String? = nil) -> String.Encoding {
+        guard let text = contentType,
+              let regex = try? NSRegularExpression(pattern: "(?<=charset=).*(?=$|\\s)"),
```

Comment:
Should probably make the regex case-insensitive and make the `.*` non-greedy in case there are other parameters.
I think `;` was a good thing to look for in addition to `$|\\s` as value-terminating characters also.